### PR TITLE
Fix Bug 1260487: Remove obsolete menu items

### PR DESCRIPTION
**Pull request by stephaniehobson**
Also fix bug 1295863: remove marketplace from nav
Also fix bug 1219354: add URL to "report content problem"

### CODE REVIEW

File: jinja2/base.html [hunk 1]
```diff
@@ -175,7 +175,7 @@
             </ul>
             <ul>
               <li><a href="{{ wiki_url('MDN/Community') }}">{{ _('Join the MDN community') }}</a></li>
-              <li><a href="https://bugzilla.mozilla.org/form.doc">{{ _('Report a content problem') }}<i aria-hidden="true" class="icon-external-link"></i></a></li>
+              <li><a href="https://bugzilla.mozilla.org/form.doc?bug_file_loc={{ request.build_absolute_uri()|urlencode }}">{{ _('Report a content problem') }}<i aria-hidden="true" class="icon-external-link"></i></a></li>
```
Comment (jwhitlock) [inline jinja2/base.html]: 💖

File: kuma/landing/jinja2/landing/homepage.html [hunk 2]
```diff
@@ -82,10 +82,8 @@ <h2 class="offscreen">{{ _('Web Technologies') }}</h2>
         <h3 class="zones"><i aria-hidden="true" class="icon-suitcase"></i><span>{{ _('Mozilla<br> Docs') }}</span></h3>
         <ul>
           <li><a href="{{ wiki_url('Mozilla/Firefox') }}">{{ _('Firefox') }}</a></li>
-          <li><a href="{{ wiki_url('Mozilla/B2G_OS') }}">{{ _('B2G OS') }}</a></li>
-          <li><a href="{{ wiki_url('Mozilla/Marketplace') }}">{{ _('Firefox Marketplace') }}</a></li>
-          <li><a href="{{ wiki_url('Apps') }}">{{ _('App Center') }}</a></li>
           <li><a href="{{ wiki_url('Mozilla/Add-ons') }}?menu">{{ _('Add-ons') }}</a></li>
```
Comment (jwhitlock) [inline kuma/landing/jinja2/landing/homepage.html]: Do you know why there is a ``?menu`` added to Add-ons?

File: jinja2/base.html [hunk 3]
```diff
@@ -159,9 +161,7 @@
             <ul>
               <li><a href="{{ wiki_url('Mozilla/Add-ons') }}">{{ _('Add-ons') }}</a></li>
               <li><a href="{{ wiki_url('Mozilla/Firefox') }}">{{ _('Firefox') }}</a></li>
-              <li><a href="{{ wiki_url('Mozilla/Marketplace') }}">{{ _('Firefox Marketplace') }}</a></li>
-              <li><a href="{{ wiki_url('Mozilla/B2G_OS') }}">{{ _('B2G OS') }}</a></li>
-              <li><a href="{{ wiki_url('Persona') }}">{{ _('Persona') }}</a></li>
+              <li><a href="{{ wiki_url('Mozilla/Add-ons/WebExtensions') }}">{{ _('WebExtensions') }}</a></li>
```
Comment (jwhitlock) [inline jinja2/base.html]: Should there be a space, like ``Web Extensions``?

File: kuma/landing/jinja2/landing/homepage.html [hunk 4]
```diff
@@ -82,10 +82,8 @@ <h2 class="offscreen">{{ _('Web Technologies') }}</h2>
...
+          <li><a href="{{ wiki_url('Mozilla/Add-ons/WebExtensions') }}">{{ _('WebExtensions') }}</a></li>
```
Comment (jwhitlock) [inline kuma/landing/jinja2/landing/homepage.html]: Same question, but I see that it is styled as ``WebExtensions`` on the documentation page too. 👽

**Comment by jwhitlock** [comment]
👍 works locally. I suspect my comments don't apply, so merge when ready.

File: kuma/landing/jinja2/landing/homepage.html (hunk 2 above)
Comment (stephaniehobson) [inline kuma/landing/jinja2/landing/homepage.html]: Nope, and I don't see any difference when I remove it.

File: jinja2/base.html (hunk 3 above)
Comment (stephaniehobson) [inline jinja2/base.html]: It's a brand like HTML5. No space: https://wiki.mozilla.org/WebExtensions

**Pull request merged**